src/main/steps/common/page.njk

```diff
@@ -1,5 +1,7 @@
 {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
 {% from "govuk/components/footer/macro.njk" import govukFooter %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
```

Comment:
@AbdullahShahHMCTS Can you please guide me what we are trying to import here ?
`{% from "govuk/components/details/macro.njk" import govukDetails %}`